### PR TITLE
add easyconfig for Python v2.7.9 with intel/2015a

### PR DESCRIPTION
**Pull request by boegel**
depends on #1288 for zlib, ncurses, libreadline, bzip2 easyconfigs

cc @hajgato, @hpcleuven, @sbecuwe

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/56f8b0a47e5d6d59a1d1 for a full test report.

**Comment by boegel** [comment]
Seems like Python builds are not a big fan of being built in directories that contain `[` and/or `]`... Trying again.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fd2d30816ef6cfd812e0 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-2.7.9-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }),
+    ('paramiko', '1.15.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/paramiko/'],
+    }),
+    ('pyparsing', '2.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyparsing/'],
+    }),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Python/Python-2.7.9-intel-2015a.eb]: move this to be next to the other dependencies.

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
remark fixed (in all Python easyconfigs), please rereview?

**Comment by wpoely86** [comment]
Looks fine

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 40 out of 51 (39 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d58e2b14311d3c1c0a1b for a full test report.

**Comment by boegel** [comment]
I consider the test report above a success: only the `iqacml` builds failed, because the test was run on a system that basically doesn't support the MPI library in that toolchain...

File: easybuild/easyconfigs/p/Python/Python-2.7.9-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        ],
+    }),
+    ('scipy', scipyversion, {
+        'source_urls': [('http://sourceforge.net/projects/scipy/files/scipy/%s' % scipyversion, 'download')],
+    }),
+    ('blist', '1.3.6', {
+        'source_urls': ['http://pypi.python.org/packages/source/b/blist/'],
+    }),
+    ('mpi4py', '1.3.1', {
+        'source_urls': ['https://mpi4py.googlecode.com/files/'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Python/Python-2.7.9-intel-2015a.eb]: moved to https://bitbucket.org/mpi4py/mpi4py

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, AMD Opteron(tm) Processor 6136, Python 2.6.6
See https://gist.github.com/8a838db3fa9b224403f0 for a full test report.

**Comment by boegel** [comment]
I'm merging this in before it gets out of control. It's synced with #1298 which is already merged.

@wpoely86: I'll fix the new source URL for mpi4py in a separate PR.

**Pull request merged**